### ISSUE TITLE
Add nether mod mapgen

### ISSUE THREAD

**Issue by dacmot** [Treer/amidstest#29]
Like hallelujah mountains, it would be great to see a map of the nether.

**Comment by JoeEnderman** [comment]
I agree. Though I understand it is a lot of extra work for not much benefit.
